runtime/caml/mlvalues.h

```diff
@@ -406,19 +438,19 @@ CAMLextern header_t *caml_atom_table;
 #define Is_none(v) ((v) == Val_none)
 #define Is_some(v) Is_block(v)
 
-/* The table of global identifiers */
-
-extern value caml_global_data;
-
 CAMLextern value caml_set_oo_id(value obj);
 
+#define Int_field(x, i) Int_val(Field(x, i))
+#define Long_field(x, i) Long_val(Field(x, i))
+#define Bool_field(x, i) Bool_val(Field(x, i))
```

Comment:
These defines are superfluous now that we don't have read barriers (the defines used to be more complex via caml_read_field).  There's nothing wrong with leaving them here, but if we prefer to minimise the size of the multicore diff then their use can be easily be reverted.

Reply:
I think we can keep them.

Reply:
Wooops, we reverted them in
https://github.com/ocaml-multicore/ocaml-multicore/pull/829
and pieces of 
https://github.com/ocaml-multicore/ocaml-multicore/pull/826

how strongly do you want them back? :-) We can restore them in a followup PR post merge pretty easily.

Reply:
I don't really want them back. What I meant was to keep them in order to save the work of removing them. If the work is done, fine.